Lib/test/test_zipimport.py

```diff
@@ -506,6 +506,32 @@ def testZipImporterMethods(self):
         self.assertEqual(zi2.archive, TEMP_ZIP)
         self.assertEqual(zi2.prefix, TESTPACK + os.sep)
 
+        # Test invalidate_caches() method
```

Comment:
This method is already egregiously long. Would you consider creating a separate method for it?